quixstreams/dataframe/dataframe.py

```diff
@@ -334,6 +342,52 @@ def test(self, value: object, ctx: Optional[MessageContext] = None) -> Any:
         composed = self.compose()
         return context.run(composed, value)
 
+    def tumbling_window(
+        self,
+        duration: Union[float, timedelta] = 30.0,
```

Comment:
Let's not set the default duration for the window. The duration must be picked manually